### ISSUE TITLE
[Accessibility] HTML element does not have lang attribute

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#837]
**Describe the bug**
HTML element does not have a lang attribute.
User Impact: **Serious**
Tool used: Axe 

**To Reproduce**
1. On the slides page, click open to view one of your uploaded slides
2. Right-click on the page and click inspect to open the devtools window
3. If you have the Axe extension installed, open it and run it
4. One of the accessibility issues discovered is that the html element does not have a lang attribute

**Expected behavior**
In keeping with the WCAG 2.1 standards for accessibility, having a lang attribute in the html element helps assistive technologies, such as screen readers, to properly interpret and pronounce text content for users who may have different language preferences or accessibility needs.

**Screenshots**
![image](https://github.com/camicroscope/caMicroscope/assets/122698422/2c58289a-3c2b-47df-a5a2-a74027a63104)

![HTML lang attribute lacking](https://github.com/camicroscope/caMicroscope/assets/122698422/e138843b-bc87-4d51-8d37-2f0d433eee74)

**Desktop:**
 - OS: Windows
 - Browser: Microsoft Edge
 - Version: Version 122.0.2365.92

**How to fix this issue**
Add the lang attribute to the <html> element of your HTML documents and specify the appropriate language code. For example, for English content, you would use lang="en", for French content, you would use lang="fr", and so on.
Ensure that the lang attribute reflects the primary language of the content on the page.

**Comment by Ayesh-07** [comment]
@nwanduka, I've made an attempt to fix this issue. I would be very appreciative if you could review it.

**Comment by nwanduka** [comment]
I'll be closing this issue now since it has been resolved in PR #856
